src/Adapter/Store/CommandHandler/BulkDeleteStoreHandler.php

```diff
@@ -53,7 +54,17 @@ public function __construct(StoreRepository $storeRepository)
     public function handle(BulkDeleteStoreCommand $command): void
     {
         foreach ($command->getStoreIds() as $storeId) {
```

Comment:
not sure if its worth blocking the PR (can be improved later), but it could use https://github.com/PrestaShop/PrestaShop/pull/31728/files#diff-7257db49bb829ae2e480f6bb8462487683c00035a33b7c56d1f145d50a35cd19R35 for bulk action

Reply:
If it's merged before it's worth using the abstract class indeed We'll see